### PR TITLE
Ajout du lien vers le blog Tech Café

### PR DESCRIPTION
**Pull request by dx44**
Site domotique généraliste avec des articles en podcast.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ Home Assistant](https://www.awesome-ha.com/) initiée et hebergé par @frenck._
 - [Projets DIY](https://projetsdiy.fr) - Plein de guide DIY avec beaucoup d'ESP, d'électronique, des raspberry et de la domotique.
 - [&lt;DEVOTICS/&gt;](https://devotics.fr/) - Blog domotique, objets connectés, nouvelles technologies, développement.
 - [figer](https://www.figer.com/) - Blog de guide et conseil en informatique, réseau et domotique.
+- [Tech Café](https://techcafe.fr/category/domotique-podcast-maison-connectee/) - Un podcast domotique généraliste avec une sensibilité Jeedom mais qui s'ouvre à HA avec un épisode spécial HA à venir.
```
Comment (oncleben31) [inline readme.md:30]:
```suggestion
- [Tech Café](https://techcafe.fr/category/domotique-podcast-maison-connectee/) - Un podcast domotique généraliste avec une sensibilité Jeedom mais qui s'ouvre à Home Assistant.
```
The incoming podcast episode information will be soon deprecated. 
I suggest we wait the episode being published and we will be able to modify the link description to advise on listening this specific episode.

**Review by oncleben31 — CHANGES_REQUESTED** [review]
Thank you some minor text changes.

File: readme.md (hunk 1 above)
Comment (dx44) [reply readme.md:30]: You're right, I pasted it too quickly whithout review the text.

Comment (oncleben31) [reply readme.md:30]: The best practice is you commit the suggestion I've made to let you have the final word on the modification in your PR.
Currently the suggestion I've made is not implemented.

**Review by oncleben31 — APPROVED** [review]

**Review by oncleben31 — APPROVED** [review]
An additional commit to move the link in a new & dedicated section for podcasts.

**Pull request merged**